demo/ops/ops.env

```diff
@@ -1,29 +1,34 @@
 # Copyright 2020 Intel Corporation
 # SPDX-License-Identifier: Apache 2.0
 
+# Add proxy and other information here.
+# Proxy to be added as below for hostname 'company-proxy.com' and ports 80 and 443:
+# -Dhttp.proxyHost=company-proxy.com -Dhttp.proxyPort=80 -Dhttps.proxyHost=company-proxy.com -Dhttps.proxyPort=443
+_JAVA_OPTIONS=
+
 # Application version used for health check request.
-application.version=1.8
+APPLICATION_VERSION=1.8
 
 # Port number at which the specific service will be listening for incoming
 # messages: Number
-server.port=8042
+SERVER_PORT=8042
 
 # Path of logback configuration file: URI
 # If not set, no configuration file will be set.
-logging.config=logback-spring.xml
+LOGGING_CONFIG=logback-spring.xml
 
 # Total number of threads the TaskExecutor is initalized with (Mandatory).
 # The default value is 10.
-thread.pool.size=10
+THREAD_POOL_SIZE=10
 
-# org.sdo.epid.epid-online-url (optional) : URL
+# (optional) : URL
```

Comment:
Property name not mentioned before optional, like it was in the earlier code

Reply:
Removed because there's no such property present in the application, and, it doesn't seem like a good idea to write the intermediate property name from .env there.

Reply:
ok